tests/functional_tests/test_container_no_root.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python3
+
+import pytest
+import testinfra
+
+from tests.functional_tests.conftest import get_pod_running_containers
+
+container_ignore_list = ["astronomer-kubed"]
+
+
+@pytest.mark.parametrize("container", get_pod_running_containers())
```

Comment:
The way the id's work here, we currently see `test_container_non_root[container6]`. It would be great if we could get the name of the container instead of the index. I know this is possible, but I'm not sure what the exact syntax would be. We don't have to fix this now, and can circle back and fix it after this first iteration, if that makes it easier to get forward progress.

Reply:
I figured it out. <https://app.circleci.com/pipelines/github/astronomer/astronomer/4270/workflows/45ee2365-42ff-46cb-b71b-4d80b4b92ae3/jobs/28388/steps>

```
=========================== short test summary info ============================
FAILED tests/functional_tests/test_container_no_root.py::test_container_non_root[astronomer-elasticsearch-exporter-59b6fdb7c8-hkvkl]
FAILED tests/functional_tests/test_container_no_root.py::test_container_non_root[astronomer-fluentd-mtvct]
FAILED tests/functional_tests/test_container_no_root.py::test_container_non_root[astronomer-houston-5bb74656bd-prpmf]
FAILED tests/functional_tests/test_container_no_root.py::test_container_non_root[astronomer-kube-state-b75f44c4d-qbh68]
```

```suggestion
containers = get_pod_running_containers()
container_ids = ids = [x["pod_name"] for x in containers]

@pytest.mark.parametrize("container", containers, ids=container_ids)
```